source/search.html

```diff
@@ -2174,51 +2253,344 @@ <h4>Prefixes</h4>
    </tr>
   <tr>
 	<td><code>eq</code></td>
-	<td>the resource value is equal to or fully contained by the parameter value</td>
-	<td>the range of the parameter value fully contains the range of the resource value</td>
+	<td>The resource value is equal to or fully contained by the parameter value.</td>
+	<td>The range of the parameter value fully contains the range of the resource value.
+      <br/><b>Boundary interpretation:</b> The lower boundary of the resource value is greater than or equal to the lower boundary of the parameter value
+       AND the upper boundary of the resource value is less than or equal to the upper boundary of the parameter value.
+      <br/><b>Resource interval tests:</b>
+      <br/>&nbsp;<code>resource-lower: [parameter-Lower, parameter-upper]</code>
+      <br/>&nbsp;<code>resource-upper: [parameter-Lower, parameter-upper]</code>
+  </td>
  </tr>
  <tr>
 	<td><code>ne</code></td>
-	<td>the resource value is not equal to the parameter value</td>
-  <td>the range of the parameter value does not fully contain the range of the resource value</td>
+	<td>The resource value is not equal to the parameter value.</td>
+  <td>The range of the parameter value does not fully contain the range of the resource value.
+      <br/><b>Boundary interpretation:</b> The lower boundary of the resource value is greater than the upper boundary of the parameter value 
+       OR the upper boundary of the resource value is less than the lower boundary of the parameter value.
+      <br/><b>Resource interval tests:</b>
+      <br/>&nbsp;<code>resource-lower: (parameter-Upper, ∞)</code>
+      <br/>&nbsp;<code>resource-upper: (-∞, parameter-lower)</code>
+  </td>

```

Comment:
Confirm the boundary and interval expressions for ne. I think this definition excludes parameter.lower < resource.lower < parameter.upper < resource.upper, which I think would be a match for the text description of ne.